### PR TITLE
add function-plot v1.13.0 

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/maurizzzio/function-plot
**Watch 9
  Star 153
 Fork 6**
@Amomo could u help me check this PR for #6139 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/function-plot/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      "name": "maurizzzio",
+      "email": "mauricio.poppe@gmail.com"
+    }
+  ],
+  "npmName": "function-plot",
+  "npmFileMap": [
+    {
+      "basePath": "/dist/",
+      "files": [
+        "function-plot*.js"
```
Comment (Amomo) [inline ajax/libs/function-plot/package.json]: Maybe we can use `**/*`

Comment (maruilian11) [inline ajax/libs/function-plot/package.json]: ooops

**Comment by maruilian11** [comment]
@Amomo i have corrected filemap :smiley:

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look? Thanks! :-D

File: ajax/libs/function-plot/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  "maintainers": [
+    {
+      "name": "maurizzzio",
+      "email": "mauricio.poppe@gmail.com"
+    }
+  ],
+  "npmName": "function-plot",
+  "npmFileMap": [
+    {
+      "basePath": "/dist/",
```
Comment (PeterDaveHello) [inline ajax/libs/function-plot/package.json]: use `dist` next time.

**Pull request merged**